integration_tests/pages/refer/programmeHistoryDetails.ts

```diff
@@ -68,4 +97,14 @@ export default class ProgrammeHistoryDetailsPage extends Page {
   shouldNotDisplayYearStartedInput() {
     cy.get('input[id="yearStarted"]').should('not.be.visible')
   }
+
+  submitDetails(courseParticipation: CourseParticipation, course: Course, person: Person) {
+    cy.task('stubUpdateParticipation', courseParticipation)
+    cy.task('stubCourse', course)
```

Comment:
Is this meant to be `stubParticipation`? I don't think we need to restub the course on submitting this form?

Reply:
stubCourse is needed as `CourseParticipationsController.courseParticipationsWithNames` calls `courseService.getCourse` so the index page doesn't load if it isn't stubbed.
<img width="724" alt="image" src="https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/assets/1067537/4de7cae6-204d-482b-9660-2f51c10ba0c5">

Reply:
Ah good point! I was reading this as just restubbing anything that's changed, rather than stubbing what will be needed on the page this takes us to